src/frontend/src/components/importer/ImportDataSelector.tsx

```diff
@@ -0,0 +1,212 @@
+import { t } from '@lingui/macro';
+import { ActionIcon, Group, Stack, Text, Tooltip } from '@mantine/core';
+import {
+  IconCircleCheck,
+  IconEdit,
+  IconExclamationCircle
+} from '@tabler/icons-react';
+import { useCallback, useMemo, useState } from 'react';
+
+import { ApiEndpoints } from '../../enums/ApiEndpoints';
+import { cancelEvent } from '../../functions/events';
+import {
+  useDeleteApiFormModal,
+  useEditApiFormModal
+} from '../../hooks/UseForm';
+import { ImportSessionState } from '../../hooks/UseImportSession';
+import { useTable } from '../../hooks/UseTable';
+import { apiUrl } from '../../states/ApiState';
+import { TableColumn } from '../../tables/Column';
+import { TableFilter } from '../../tables/Filter';
+import { InvenTreeTable } from '../../tables/InvenTreeTable';
+import { RowDeleteAction, RowEditAction } from '../../tables/RowActions';
+import { ApiFormFieldSet } from '../forms/fields/ApiFormField';
+
+function ImporterDataCell({
+  column,
+  row,
+  onEdit
+}: {
+  column: any;
+  row: any;
+  onEdit?: () => void;
+}) {
+  const onRowEdit = useCallback(
+    (event: any) => {
+      cancelEvent(event);
+      onEdit?.();
+    },
+    [onEdit]
+  );
+
+  const cellErrors: string[] = useMemo(() => {
+    return row?.errors[column.field] ?? [];
+  }, [row.errors, column.field]);
+
+  return (
+    <Group grow position="apart">
+      <Group grow style={{ flex: 1 }}>
+        <Stack spacing="xs">
+          <Text>{row.data[column.field]}</Text>
+          {cellErrors.map((error: string) => (
+            <Text size="xs" color="red">
```

Comment:
Don't forget to add a key when rendering elements in a loop.

```suggestion
            <Text size="xs" color="red" key={error}>
```